Source/WebCore/platform/network/curl/CurlRequest.cpp

```diff
@@ -396,7 +396,14 @@ void CurlRequest::didReceiveHeaderFromMultipart(const Vector<String>& headers)
     for (auto header : headers)
         response.headers.append(header);
 
-    invokeDidReceiveResponse(response);
+    invokeDidReceiveResponse(response, [this] {
```

Comment:
Do you need `protectedThis` here?

Reply:
Not needed as it is set in CurlRequest::callClient.